src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -154,6 +154,7 @@ private void getNotifyBoth(boolean self, String query, int offset, int limit,
       }
       cql = getCQL(query, limit, offset);
     } catch (Exception e) {
+      logger.warn("getNotifyBoth:: exception: " + e.getMessage());
```

Comment:
As it says in the docs, exceptions should always be logged with stack trace (in this case, `e` should be the last param). Also, formatting should be used instead of string concatenation.